packages/access-client/src/agent.js

```diff
@@ -642,23 +763,28 @@ export class Agent {
   async invoke(cap, options) {
     const space = options.with || this.currentSpace()
     if (!space) {
-      throw new Error('No space selected, you need pass a resource.')
+      throw new Error(
+        'No space or resource selected, you need pass a resource.'
+      )
     }
 
+    const extraProofs = options.proofs || []
     const proofs = this.proofs([
       {
         with: space,
         can: cap.can,
       },
     ])
```

Comment:
```suggestion
    const proofs = [
      ...(options.proofs || []),
      ...this.proofs([
      {
        with: space,
        can: cap.can,
      },
    ])
 ]
```
Might be easier to read

Reply:
@travis